src/services/routingSlip.services.ts

```diff
@@ -4,18 +4,19 @@ import axios from '@/util/http-util'
 
 export default class RoutingSlip {
   public static async getRoutingSlip (
-    routingNumber: string
+    routingNumber: string,
+    showLoader: boolean = false
   ): Promise<AxiosResponse> {
     return axios.get(
-      `${ConfigHelper.getFasAPIURL()}/routing-slips/${routingNumber}`
+      `${ConfigHelper.getFasAPIURL()}/routing-slips/${routingNumber}`, { showLoader: showLoader }
     )
   }
 
   public static async createRoutingSlip (
     routingSlipRequest: RoutingSlip
   ): Promise<AxiosResponse> {
     return axios.post(
-      `${ConfigHelper.getFasAPIURL()}/routing-slips`, routingSlipRequest
+      `${ConfigHelper.getFasAPIURL()}/routing-slips`, routingSlipRequest, { showLoader: true }
```

Comment:
better to pass showLoader from action like in line 8

Reply:
yes on it :)